src/tests/ftest/container/simple_create_delete_test.py

```diff
@@ -51,48 +52,32 @@ def test_container_basics(self):
         Test basic container create/destroy/open/close/query.  Nothing fancy
         just making sure they work at a rudimentary level
 
-        :avocado: tags=container,containercreate,containerdestroy,basecont
+        :avocado: tags=pr,container,containercreate,containerdestroy,basecont
         """
-
-        pool = None
-        hostlist = None
-
         try:
-            hostlist = self.params.get("test_machines", '/run/hosts/*')
-            hostfile = write_host_file.write_host_file(hostlist,
-                                                       self.workdir)
+            self.hostlist_servers = self.params.get("test_machines", '/run/hosts/*')
+            self.hostfile_servers = write_host_file.write_host_file(
+                                    self.hostlist_servers, self.workdir)
 
-            self.agent_sessions = agent_utils.run_agent(self.basepath, hostlist)
-            server_utils.run_server(hostfile, self.server_group, self.basepath)
+            self.agent_sessions = run_agent(self.basepath, self.hostlist_servers)
+            run_server(self.hostfile_servers, self.server_group, self.basepath)
 
             # give it time to start
             time.sleep(2)
```

Comment:
We can also remove sleep() here. server_utils.py takes care if servers are started properly.

Reply:
Will remove.